okapi-test-auth-module/src/main/java/org/folio/okapi/auth/Auth.java

```diff
@@ -222,6 +223,8 @@ public void check(RoutingContext ctx) {
     }
     // Fake some module tokens
     String modTok = moduleTokens(ctx);
+    FolioLoggingContext.put(FolioLoggingContext.USER_ID_LOGGING_VAR_NAME,
+        userId);
```

Comment:
As chatted, we do not need to change okapi-test-auth-module/src/main/java/org/folio/okapi/auth/Auth.java